### ISSUE TITLE
UX issues with markdown editor

### ISSUE THREAD

**Issue by barefoot-88** [peakshift/makers.bolt.fun#33]
Whilst uploading a new story to the makers platform, I noticed a few issues with the platforms markdown editor. Here are some thoughts and potential ideas of how we could improve them (this might be limited by the markdown integration we use).

a). Every time I want to add a title I have to highlight the word and scroll to the top, click T... it then takes me back down to where I was typing. This was annoying when I wanted to change an Italicised word to only Bold. I had to highlight it, scroll up, click off 'Italic', it then shot me back down the page, I had to highlight it again, scroll up to the top and click 'Bold' ... not so annoying when it's only once but when it's over and over it could get frustrating

**How can we solve this?** 
It seems like the easiest way to solve this might be to fix the height of the ui component which contains the bulk of the story text, whilst fixing the markdown shortcuts to the top of it (e.g H B _i_ ) .. I have attached a screenshot of the Github markdown editor for reference! If the user then has their mouse within the text box and scrolls, they can scroll the content of their writing easily without having to constantly go to the top of the website page and back over and over. 

<img width="916" alt="Screenshot 2022-06-13 at 13 18 11" src="https://user-images.githubusercontent.com/36778205/173285346-f77a6ddb-8294-4e0d-b8ee-171af586e210.png">

This second point is to do with editing functionality after a post has been created...

b). When I clicked 'Edit post' and then saved it, it actually created a duplicate of the post! This seems like a backend issue rather than UI

**Comment by johnsBeharry** [comment]
I'd propose we do a sticky toolbar. https://dev.to also does something similar.

![sticky-toolbar](https://user-images.githubusercontent.com/183140/173287669-6ba1982c-e6f2-4ac0-a74f-9057ee9ea827.gif)
